.github/workflows/test.yml

```
@@ -281,3 +281,80 @@ jobs:
 
           pip install -r requirements-test.txt
           pytest
+  build-and-test-java-api:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: windows-latest
+          - os: macos-latest
+          - os: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Rust
+        uses: ./.github/actions/rust-toolchain-from-file
+      - name: Set up Java
+        uses: actions/setup-java@v2
+        with:
+          java-version: "11"
+          distribution: "adopt"
+      - name: Build
+        run: |
+          cargo build -p voicevox_core_java_api -vv
+          cargo build -p test_util -vv
+      - name: 必要なDLLをコピーしてテストを実行
+        working-directory: crates/voicevox_core_java_api
+        run: |
+          ARCH="$(uname -m)"
+          case "${{ matrix.os }}" in
+          windows-latest)
+            DLL_NAME="voicevox_core_java_api.dll"
+            case "$ARCH" in
+              x86_64)
+                TARGET_NAME="windows-x64"
+                ;;
+              *)
+                echo "Unsupported architecture: $ARCH"
+                exit 1
+                ;;
+            esac
+            ;;
+          macos-latest)
+            DLL_NAME="libvoicevox_core_java_api.dylib"
+            case "$ARCH" in
+              x86_64)
+                TARGET_NAME="mac-x64"
+                ;;
+              arm64)
+                TARGET_NAME="mac-arm64"
+                ;;
+              *)
+                echo "Unsupported architecture: $ARCH"
+                exit 1
+                ;;
+            esac
+            ;;
+          ubuntu-latest)
+            DLL_NAME="libvoicevox_core_java_api.so"
+            case "$ARCH" in
+              x86_64)
+                TARGET_NAME="linux-x64"
+                ;;
+              aarch64)
+                TARGET_NAME="linux-arm64"
+                ;;
+              *)
+                echo "Unsupported architecture: $ARCH"
+                exit 1
+                ;;
+            esac
+            ;;
+          *)
+            echo "Unsupported OS: ${{ matrix.os }}"
+            exit 1
+            ;;
+          esac
+          cp -v ../../target/debug/$DLL_NAME ./lib/src/main/resources/dll/$TARGET_NAME/$DLL_NAME
```

```suggestion
          cp -v "../../target/debug/$DLL_NAME" "./lib/src/main/resources/dll/$TARGET_NAME/$DLL_NAME"
```

[SC2086](https://www.shellcheck.net/wiki/SC2086)に違反しており、本来ならactionlintに引っ掛かっていると思います。
**この上の`exit 1`の部分を抜いたら**actionlintがちゃんと反応し始めました。何故...???